tests/FunctionalServerTest.php

```diff
@@ -479,12 +479,15 @@ public function testRequestHandlerWithStreamingRequestWillReceiveCloseEventIfCon
         $connector = new Connector($loop);
 
         $once = $this->expectCallableOnce();
-        $server = new Server(array(
-            new StreamingRequestMiddleware(),
-            function (RequestInterface $request) use ($once) {
-                $request->getBody()->on('close', $once);
-            }
-        ));
+        $server = new Server(
+            $loop,
```

Comment:
Indentation is *okay* as-is, but could also be left unchanged because follow-up #362 will remove the `array` indentation level anyway (to keep diffs smaller).

Reply:
Yeah started doing this, but half way stopped to keep this simpler. Reverting this now